srcpkgs/jose/template

```diff
@@ -0,0 +1,66 @@
+# Template file for 'jose'
+pkgname=jose
+version=14
+revision=1
+build_style=meson
+configure_args="-Dwerror=false"
+hostmakedepends="asciidoc pkg-config"
+makedepends="jansson-devel openssl-devel zlib-devel"
+short_desc="C-language implementation of Javascript Object Signing and Encryption"
+maintainer="Johannes Heimansberg <git@jhe.dedyn.io>"
+license="Apache-2.0"
+homepage="https://github.com/latchset/jose"
+distfiles="https://github.com/latchset/jose/releases/download/v$version/jose-$version.tar.xz"
+checksum=cee329ef9fce97c4c025604a8d237092f619aaa9f6d35fdf9d8c9052bc1ff95b
+
+# The -Db_ndebug=true flag from the default meson do_configure causes a test to fail,
```

Comment:
can you pass it in configure_args? i would think a flag passed twice would take the later value. if that doesn't work, i'd rather just `make_check=no # tests need -Db_ndebug=true` than copy a section of the build style